src/accessories/DimmerAccessory.ts

```diff
@@ -42,4 +43,19 @@ export class DimmerAccessory extends BaseAccessory {
 
     return super.deviceSupportedCharacteristics;
   }
+
+  validateConfigOverwrites(config: TuyaDeviceDefaults): string[] {
+    const errors = super.validateConfigOverwrites(config);
+    if (config?.max_brightness) {
+      const maxBrigthness = config.max_brightness;
```

Comment:
This looks odd to me. We are getting config.max_brightness to only reassign it back to its original value, if it was valid? What's the purpose of that? Same in the other file?

Reply:
We are missing a cast to number here.
Most config variables are or can be strings, so we need to use
`const maxBrightness = Number(config.max_brightness)`

That also explains why we write it back.